demos/object_detection_demo/cpp/main.cpp

```diff
@@ -311,28 +314,32 @@ int main(int argc, char *argv[]) {
         }
 
         InferenceEngine::Core core;
+
         AsyncPipeline pipeline(std::move(model),
             ConfigFactory::getUserConfig(FLAGS_d, FLAGS_l, FLAGS_c, FLAGS_pc, FLAGS_nireq, FLAGS_nstreams, FLAGS_nthreads),
-            core);
+            core
+            );
         Presenter presenter(FLAGS_u);
 
         bool keepRunning = true;
         int64_t frameNum = -1;
         std::unique_ptr<ResultBase> result;
         uint32_t framesProcessed = 0;
+
         cv::VideoWriter videoWriter;
+        cv::Size frameSize;
+        double fps = 30;
```

Comment:
Why are `frameSize` and `fps` stored now?

Reply:
that's for videoWriter. I've changed then to videoFrameSize and videFps to avoid confusion

Reply:
`videoWriter` doesn't need them to be stored in `main()`.

Reply:
It is used in videoWriter.open() function in the code below. What's wrong ? Do you have fear of extra local variable ?

Reply:
> What's wrong ? Do you have fear of extra local variable ?

You wrote it yourself: the variable is extra.